WhaleSpotting/react-app/src/styles/Navbar.scss

```diff
@@ -9,38 +9,127 @@
     justify-content: center;
     align-items: center;
     height: $headerHeight;
-}
 
-.nav-bar {
-    display: flex;
-    width: 100%;
-    max-width: 1024px;
-    flex-direction: row;
-    justify-content: space-between;
-}
-
-.navbar-link {
-    margin: 0 5%;
-    padding: 3%;
-    white-space: nowrap;
-    color: $textOnPrimary;
-    border-radius: 5px;
-    text-decoration: none;  
-    font-size: $bodyFontSize;  
-
-    &.selected, &:hover {
-        background-color: $primaryDark;
-        text-decoration: none;
+
+    .nav-bar {
+        display: flex;
+        width: 100%;
+        max-width: 1024px;
+        flex-direction: row;
+        justify-content: space-between;
+    }
+
+    .navbar-link {
+        margin: 0 5%;
+        padding: 3%;
+        white-space: nowrap;
         color: $textOnPrimary;
-        border-radius: 5px; 
+        border-radius: 5px;
+        text-decoration: none;  
+        font-size: $bodyFontSize;  
+
+        &.selected, &:hover {
+            background-color: $primaryDark;
+            text-decoration: none;
+            color: $textOnPrimary;
+            border-radius: 5px; 
+            width: max-content;
+        }
+    }
+
+    .fixed-nav-links {
+        justify-self: flex-start;
     }
-}
 
-.fixed-nav-links {
-    justify-self: flex-start;
-}
+    .changing-nav-links {
+        justify-self: flex-end;
+        margin-right: 5%;
+    }
+
+    .burger-button {
+        display: none;
+        cursor: pointer;
+        position: absolute;
+        margin: 0 6px;
+        width: 30px;
+        right: 20px;
+        z-index: 3;
+        background-color: $primaryColour;
+    }
+
+    .current-page-mobile {
+        display: none;
+    }
+
+    @mixin bar {
+        width: 35px;
+        height: 5px;
+        background-color: $secondaryLight;
+        margin: 6px 0;
+        transition: 0.4s;
+    }
 
-.changing-nav-links {
-   justify-self: flex-end;
-   margin-right: 5%;
+    .opened-bar-1, .opened-bar-2, .opened-bar-3 {
+        @include bar;
+    }
+
+    /* Rotate first bar */
+    .closed-bar-1 {
+        @include bar;
+        -webkit-transform: rotate(-45deg) translate(-8px, 7px);
+        transform: rotate(-45deg) translate(-8px, 7px);
+    }
+
+    /* Fade out the second bar */
+    .closed-bar-2 {
+        @include bar;
+        opacity: 0;
+    }
+
+    /* Rotate last bar */
+    .closed-bar-3 {
+        @include bar;
+        -webkit-transform: rotate(45deg) translate(-8px, -8px);
+        transform: rotate(45deg) translate(-8px, -8px);
+    }
+    
+    @media(max-width: $mobileScreenSize) {
+
+        justify-content: left;
+        padding-left: 5px;
```

Comment:
This padding also makes the menu now not cover the whole page, which I am not sure is what we want?

Reply:
adjusted - margin-left now added to current-page-mobile
![image](https://user-images.githubusercontent.com/84725775/128182029-d87d69ce-f072-43cc-a26e-8caec6a047b0.png)
![image](https://user-images.githubusercontent.com/84725775/128182061-6ca1e011-289a-462e-b180-9bc74380d042.png)